schema.ts

```diff
@@ -356,29 +370,75 @@ export enum ScaleIconSet {
   LightBulb = 'lightbulb',
 }
 
+/**
+ * A schema representing a return value or object property that is a number that should
+ * be rendered as a scale.
+ *
+ * A scale is a widget with a repeated set of icons, where the number of shaded represents
+ * a numeric value. The canonical example of a scale is a star rating, which might show
+ * 5 star icons, with 3 of them shaded, indicating a value of 3.
+ */
 export interface ScaleSchema extends BaseNumberSchema<ValueHintType.Scale> {
+  /** Instructs Coda to render this value as a scale. */
   codaType: ValueHintType.Scale;
+  /** The number of icons to render. */
   maximum?: number;
+  /** The icon to render. */
   icon?: ScaleIconSet;
 }
 
+/**
+ * A schema representing a return value or object property that is provided as a string,
+ * which Coda should interpret as a date. Coda is able to flexibly a parse number of formal
+ * and informal string representations of dates. For maximum accuracy, consider using an
+ * ISO 8601 date string (e.g. 2021-10-29): https://en.wikipedia.org/wiki/ISO_8601.
+ */
 export interface StringDateSchema extends BaseStringSchema<ValueHintType.Date> {
+  /** Instructs Coda to render this value as a date. */
   codaType: ValueHintType.Date;
-  // A Moment date format string, such as 'MMM D, YYYY', that corresponds to a supported Coda date column format.
+  /**
+   * A Moment date format string, such as 'MMM D, YYYY', that corresponds to a supported Coda date column format.
```

Comment:
Is it worth clarifying that this is the output format, not the input format? Ditto for Time and DateTime below.

Reply:
Sure, updated all usage.